src/llir/raise/early.rs

```diff
@@ -665,7 +679,7 @@ impl AtomRaiser<'_, '_> {
             | ArgEncoding::Integer { ty_color: None, format, .. }
             => Ok(ast::Expr::LitInt { value: raw.expect_int(), format: *format }),
 
-            | ArgEncoding::Padding
+            | ArgEncoding::Padding { .. }
             => Ok(ast::Expr::from(raw.expect_int())),
```

Comment:
I wonder if this is unreachable now?

Reply:
It's definitely not unreachable since the padding args are only dropped after being raised thanks to some sort of logic with `@arg0`. I don't know how important the positioning of that is since you'd left a comment about it being important to reference the original args list, so I chose to leave it there for now. If it's possible to implement the "verify the bytes are 0 or fallback to blob" check at at line 541 before raising all the arguments, then it would be unreachable for sure.